racksdb/web/ui/src/views/DatacenterDetailsView.vue

```diff
@@ -50,66 +75,82 @@ const props = defineProps({
 <template>
   <BreadCrumbs :datacenterName="props.name" />
 
-  <SearchBar
-    v-if="datacenters.length"
-    viewTitle="Datacenter Details"
-    searchedItem="datacenter"
-    :items="datacenters"
-  />
+  <div class="flex justify-end items-center p-4 px-20">
+    <input
+      type="text"
+      v-model="input"
+      class="w-96 rounded-md focus:border-purple-700 focus:outline-none border-2 border-solid bg-white py-1.5 pl-3 pr-10 text-gray-900 shadow-sm ring-1 ring-inset ring-gray-300 sm:text-sm sm:leading-6"
+      :placeholder="`Filter room by name`"
+      required
+    />
+  </div>
 
-  <div v-if="datacenterDetails">
+  <div v-if="datacenterDetails" class="flex justify-center my-auto mx-auto">
```

Comment:
`datacenterDetails` is not used anymore in this template, you should better check if `filteredRooms` is not undefined here.

Reply:
Right, I corrected it.